qemu/tests/cfg/rh_kselftests_vm.cfg

```diff
@@ -0,0 +1,10 @@
+- rh_kselftests_vm:
+    only RHEL
+    type = rh_kselftests_vm
+    virt_test_type = qemu
+    kernel_path = "/home/kernel"
```

Comment:
Suggest use the `/tmp` dir, and clean up after test finish.

Reply:
Updated !